libraries-http-2/pom.xml

```diff
@@ -120,6 +120,7 @@
         <reactive.stream.version>1.0.3</reactive.stream.version>
         <reactor.version>3.6.0</reactor.version>
         <jmockit.version>1.49</jmockit.version>
+        <mockito.version>5.2.0</mockito.version>
```

Comment:
Shouldn't this be 5.11.0?

Reply:
The latest version for [mockito-inline](https://mvnrepository.com/artifact/org.mockito/mockito-inline/5.2.0) is 5.2.0, hence the version has been forced in some child modules.